skater/player.py

```diff
@@ -102,40 +103,41 @@ def move(self, screen, event, gameboard):
     
     def call_movement_functions(self, gameboard):
         self.move_x(gameboard)
-        #self.check_crash()
+        self.move_y(gameboard)
         self.handle_images()
 
 
     def move_x(self, gameboard):
+        # find x position of the closest obstacle edges on the right and left side of the player
+        # NOTE: this has to be computed *before* modifying self.rect.x
+        limit_right = gameboard.limit_right(self)
```

Comment:
To powodowało błędy -> limit był wyliczany na podstawie złych wartości.

Reply:
ah makes sense superko dzieki